readme_pages/pylibcugraph.md

```diff
@@ -0,0 +1,25 @@
+<h1 align="center";>
+  <br>
+  <img src="../img/cugraph_logo_2.png" alt="cuGraph" width="300">
+</h1>
+<h1 align="left";>
+  <br>
+CuGraph pylibcugraph
+</h1>
+
+Part of [Rapids](https://rapids.ai) cuGraph, pylibcugraph is a wrapper around the cuGraph C API. It is aimed more at integrators instead of algorithm writers or end users like Data Scientists. Much of the cuGraph python API relies on pylibcugraph to efficiently run algorithms removes much of the overhead of more python-centric implementation relying more on cython.Pylibcugraph is intended for applications that require a tighter integration with cuGraph at the Python layer with fewer dependencies.
```

Comment:
```suggestion
Part of [RAPIDS](https://rapids.ai) cuGraph, pylibcugraph is a wrapper around the cuGraph C API. It is aimed more at integrators instead of algorithm writers or end users like Data Scientists. Much of the cuGraph python API relies on pylibcugraph to efficiently run algorithms removes much of the overhead of more python-centric implementation relying more on cython.Pylibcugraph is intended for applications that require a tighter integration with cuGraph at the Python layer with fewer dependencies.
```

Reply:
Fixed